graph/step.go

```diff
@@ -51,10 +51,13 @@ type Step struct {
 	Network                         string   `yaml:"network"`
 	Isolation                       string   `yaml:"isolation"`
 	IgnoreErrors                    bool     `yaml:"ignoreErrors"`
-	Retries                         int      `yaml:"retries"`
 	RetryDelayInSeconds             int      `yaml:"retryDelay"`
 	DisableWorkingDirectoryOverride bool     `yaml:"disableWorkingDirectoryOverride"`
 	Pull                            bool     `yaml:"pull"`
+	// Retries specifies how many times a Step will be retried if it fails after its initial execution.
+	Retries int `yaml:"retries"`
```

Comment:
I assume the code checks retries and repeats are >0 ?

Reply:
Added in validation. It'll throw an error now.